tests/specs/IgnoreComments/IgnoreComments_All.txt

```diff
@@ -1,45 +1,46 @@
+~~ ignoreDirective: foo-ignore, ignoreStartDirective: foo-ignore-start, ignoreEndDirective: foo-ignore-end ~~
```

Comment:
Looks great! Just swap the file names and we're good (IgnoreComments_All <-> IgnoreComments_Custom)

Reply:
Oooops, my bad